ecosystem-modules/ren/renvm-bridge/src/mock.rs

```diff
@@ -4,7 +4,7 @@
 
 use super::*;
 use frame_support::{impl_outer_event, impl_outer_origin, parameter_types};
-use orml_currencies::BasicCurrencyAdapter;
+use module_currencies::BasicCurrencyAdapter;
```

Comment:
no need to replace those in mocks unless you are explicitly testing if this pallet works with module-currencies